src/NServiceBus.Core/Transports/ErrorContext.cs

```diff
@@ -46,5 +71,10 @@ public ErrorContext(Exception exception, Dictionary<string, string> headers, str
         /// Failed incoming message.
         /// </summary>
         public IncomingMessage Message { get; }
+
+        /// <summary>
+        /// A <see cref="ContextBag" /> which can be used to extend the current object.
```

Comment:
This description might not be accurate given that the consumer of the ErrorContext can't extend anything on a read-only context bag. Maybe change this to something about additional information provided by the transport?